packages/gestalt-codemods/generic-codemods/renameComponent.js

```diff
@@ -0,0 +1,82 @@
+// @flow strict
+
+/**
+ * CODEMOD to RENAME GESTALT COMPONENTS
+ * Ex. <Flyout /> to <Popover />
+ *
+ * TO RUN THIS CODEMOD
+ * yarn run:codemod renameComponent  <folder/file path> --previousCmpName=<value> --nextCmpName=<value>
```

Comment:
It looks like this command needs to be updated? `yarn codemod ...`

Reply:
DONE